TableDeck.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Card_Game
+{
+    public class TableDeck : Deck
+    {
+        private ICardsDAO FullDeck { get; set; }
+        private Random Rand = new Random();
```

Comment:
Can we move Random to method that uses it? There is no need to keep it as field in class.